docs/api/public_php_api_browsing.md

```diff
@@ -1,347 +1,340 @@
+    $output->write("Version $versionInfo->versionNo ");
+    $output->write(" by " . $creator->contentInfo->name);
+    $output->writeln(" " . $this->outputStatus($versionInfo->status) . " " . $versionInfo->initialLanguageCode);
 }
 ```
 
-As said earlier, the public API uses [Exceptions](http://php.net/exceptions) to handle errors. Each method of the API may throw different exceptions, depending on what it does. Which exceptions can be thrown is usually documented for each method. In our case, `loadContent()` may throw two types of exceptions: `NotFoundException`, if the requested ID isn't found, and `UnauthorizedException` if the currently logged in user isn't allowed to view the requested content.
+You can additionally provide the `loadVersions()` method with the version status
+to get only versions of a specific status, e.g.:
 
-It is a good practice to cover each exception you expect to happen. In this case, since our Command takes the Content ID as a parameter, this ID may either not exist, or the referenced Content item may not be visible to our user. Both cases are covered with explicit error messages.
-
-### SiteAccess-aware Repository - optional
+``` php
+$versionInfoArray = $this->contentService->loadVersions($contentInfo, VersionInfo::STATUS_DRAFT);
+```
 
-The SiteAccess-aware repository is an instance of the eZ Platform Repository API which injects prioritised languages for you when you load data *(Content, Location, Content type, etc.)* and don't specify languages to load yourself in API arguments.
+!!! note
 
-Currently it is:
+    Requesting Version data may be impossible for an anonymous user.
+    Make sure to [authenticate](public_php_api.md#setting-the-repository-user) as a user with sufficient permissions.
 
-- Available as a private service `ezpublish.siteaccessaware.repository`, including for the other Repository services
-- Used out of the box on parameter converters for Content and Location
-- Used out of the box on ContentView
+### Relations
 
-Example of code before using SiteAccess-aware repository:
+Content Relations are versioned.
+To list Relations to and from your Content,
+you need to provide the [`ContentService::loadRelations()`](https://github.com/ezsystems/ezpublish-kernel/blob/v7.5.2/eZ/Publish/API/Repository/ContentService.php#L347) method with a `VersionInfo` object.
+You can get the current version's `VersionInfo` using [`ContentService::loadVersionInfo()`](https://github.com/ezsystems/ezpublish-kernel/blob/v7.5.2/eZ/Publish/API/Repository/ContentService.php#L80).
 
 ``` php
-$content = $this->contentService->loadContent(
-    42,
-    $this->configResolver->getParameter('languages')
-);
-
-$name = $content->getVersionInfo()->getName();
-$value = $content->getFieldValue('body')
+$versionInfo = $this->contentService->loadVersionInfo($contentInfo);
+$relations = $this->contentService->loadRelations($versionInfo);
+if (count($relations)) {
+    foreach ($relations as $relation) {
```

Comment:
`if` statement is not needed - `foreach` won't loop through the elements if passed array is empty